tensorflow/python/ops/embedding_ops.py

```diff
@@ -307,6 +307,10 @@ def embedding_lookup(
       element must be appropriately sized for the given `partition_strategy`.
     ids: A `Tensor` or a 'RaggedTensor' with type `int32` or `int64` containing
       the ids to be looked up in `params`.
+      
+      Caution: On CPU, if an out of bound id is found, an error is raised.
```

Comment:
Is this true for XLA as well?

Reply:
@tf-security-autoupdate-bot @tensorflow-jenkins @googlebot @snyk-bot

Reply:
> @tf-security-autoupdate-bot @tensorflow-jenkins @googlebot @snyk-bot 

@davidmerwin @tensorflow-copybara

Reply:
> Is this true for XLA as well?

@cantonios ,

With XLA, all out of bound `ids` are replaced with **maximum** of `ids` inferred from the input `params`.Both xla-cpu and xla-gpu behaviour is same.But entirely different from non-xla results. Please refer attached [gist](https://colab.sandbox.google.com/gist/SuryanarayanaY/98ae4c2a85e4f3c7cac58bf8cddc1737/62628.ipynb).

Reply:
Right, so this added comment is not always true.  Please update the comment then.

Reply:
@cantonios , Updated the XLA behaviour as suggested. Thanks!